pyproject.toml

```diff
@@ -5,7 +5,11 @@ description = "Automates information gathering, vulnerability scanning and aids
 license = "Apache-2.0"
 readme = "README.md"
 
-authors = ["OWASP Nettacker Contributors"]
+authors = ["OWASP Nettacker Contributors <nettacker-project@owasp.org>"]
+homepage = "https://owasp.org/nettacker"
```

Comment:
This would look more consistent w/ the docs URL. I'll leave it to you to decide which one is better.
```suggestion
homepage = "https://nettacker.owasp.org"
```